components/src/atoms/dropdown-list-link/DropdownListLink.html

```diff
@@ -0,0 +1,12 @@
+<li class="dropdown-list__item">
```

Comment:
I know it looks like an atom, but as far as it's a part of `DropdownList` component I think it should be moved to `/molecules/dropdown-list/link/`, the same for `DropdownListItem` - can be moved to `/molecules/dropdown-list/item/`. I think we should group components in this way, they also have shared styles in main component.
cc @szafran89 what do you think?

Reply:
Initially it was done in this way, but later we assumed that everything should be on one level. 
In alpaca channel was wrote:
> one component - one directory, now we have some sets of components like table, dropdown-list etc.
But maybe we should discuss it again.

Reply:
Ok, let's stay with this convention for now